en/simulation/multi-vehicle-simulation.md

```diff
@@ -19,14 +19,13 @@ To test an example setup, follow the below steps,
   ```
   cd Firmware_clone
   git submodule update --init --recursive
-  make posix_sitl_default gazebo
+  make posix_sitl_default sitl_gazebo
   ```
 * source your environment,
```

Comment:
These bullets should be numbered items (all items can be `1. ` in markdown) if the steps must be followed in order.